azure-iot-device/tests/iothub/pipeline/test_pipeline_stages_iothub_mqtt.py

```diff
@@ -226,6 +227,42 @@ def set_connection_args_for_module(set_connection_args):
 
 
 class IoTHubMQTTConverterStageTestBase(StageTestBase):
+
+    # This is initially defined in helpers.py under the common pipeline tests,
+    # However that definition is for IoTHub specifically, because we need to
+    # make sure that root is configured using the IoTHubPipelineConfig.
+    # YMTODO: The comment below states that it's not guaranteed to be called
+    # before other fixtures run. Because of that, maybe I need to do something different...
```

Comment:
This doesn't matter. 
I think that "not guaranteed" comment is a little esoteric because it's much more specific than I had thought at the time.  Because this fixture is autouse, it gets applied before any fixtures that are not autouse.  However, we can't guarantee that it gets applied before any other autouse fixtures, but I don't think that's going to be a problem in this particular case.    I would remove this comment and the one below.  (and the source comment too).